core/domain/src/main/java/com/pronoidsoftware/core/domain/agendaitem/AgendaItem.kt

```diff
@@ -68,7 +68,13 @@ data class Attendee(
     val remindAt: LocalDateTime,
 )
 
-data class Photo(
-    val key: String,
-    val url: String,
-)
+sealed interface Photo {
+    data class Remote(
+        val key: String,
+        val url: String,
+    ) : Photo
+
+    data class Local(
+        val compressedPhotoUri: String,
+    ) : Photo
```

Comment:
Quite an edge case, but it seems you use the local Uri as an identifier for one photo. What happens if the user adds the same photo twice, then I think they will have the same identifier which could break the app, if you're using lazy column keys. Also deleting one of them will delete both